src/utils/platform.rs

```diff
@@ -1,22 +1,25 @@
 use std;
 
-#[cfg(not(target_os = "macos"))]
+#[cfg(not(any(target_os = "macos", target_os = "linux")))]
 pub fn get_block_size() -> u64 {
     1024
 }
 
-#[cfg(target_os = "macos")]
+#[cfg(any(target_os = "macos", target_os = "linux"))]
 pub fn get_block_size() -> u64 {
     512
 }
 
 #[cfg(target_os = "linux")]
-pub fn get_metadata(d: &std::fs::DirEntry, s: bool) -> Option<(u64, Option<(u64, u64)>)> {
+pub fn get_metadata(
+    d: &std::fs::DirEntry,
+    use_apparent_size: bool,
+) -> Option<(u64, Option<(u64, u64)>)> {
     use std::os::linux::fs::MetadataExt;
     match d.metadata().ok() {
         Some(md) => {
             let inode = Some((md.st_ino(), md.st_dev()));
-            if s {
+            if use_apparent_size {
```

Comment:
The variable name 's' kept the function def on one line. I'm not sure that this is a valid reason to keep a bad name like 's' but just letting you know why it was 's'.

I'm happy with either version.